docker-compose.gha.yml

```diff
@@ -83,6 +83,8 @@ services:
       - NODE_ENV=${NODE_ENV}
       - RATE_LIMIT_THRESHOLD
       - UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH
+      - CONTACT_US_SELF_SERVICE_PORTAL_URL
```

Comment:
Does CONTACT_US_EMAIL_ADDRESS needs to be added to 'portal' on all the docker compose please?

Reply:
Please ignore have noticed portal PR